packages/enzyme-test-suite/test/ReactWrapper-spec.jsx

```diff
@@ -1526,6 +1528,123 @@ describeWithDOM('mount', () => {
         expect(wrapper.find(Component.displayName)).to.have.lengthOf(2);
       });
     });
+
+    
+    describe('find DOM elements by constructor', () => {
```

Comment:
i would do something like:
```suggestion
      describeWithDOM('find DOM elements by constructor', () => {
```
(import it from `'./_helpers'`) and it will skip the test for you when there's no DOM available.